### PR TITLE
[ci skip] Minor documentation fixes for consistency

### PR DESCRIPTION
**Pull request by shailesh-kalamkar**

### CODE REVIEW

**Comment by shailesh-kalamkar** [comment]
This PR fixes minor documentation fixes for consistency- We are consistently using the word `test case` and not `test-case`. Same is the case with the word `use case`.

**Comment by fxn** [comment]
Yeah, "use case" is better written that way. 👍

File: actionview/test/ujs/public/vendor/jquery-2.2.0.js [hunk 1]
```diff
@@ -9639,7 +9639,7 @@ jQuery.fn.extend( {
 	// 2) For the hidden or detached element
 	// 3) For body or html element, i.e. in case of the html node - it will return itself
 	//
-	// but those exceptions were never presented as a real life use-cases
+	// but those exceptions were never presented as a real life use cases
```
Comment (sharang-d) [inline actionview/test/ujs/public/vendor/jquery-2.2.0.js]: This is the jquery source file. We should not be editing it IMO.

Comment (fxn) [reply actionview/test/ujs/public/vendor/jquery-2.2.0.js]: Oh, that was an overlook, didn’t realize it.

Don’t have a computer this weekend, would you mind opening a PR reverting this edit? Please cc me if you do.

**Comment by shailesh-kalamkar** [comment]
Thanks for pointing it out and the fix. Need to be careful in the future.

**Pull request merged**